src/test/java/org/opentripplanner/middleware/ApiKeyManagementTest.java

```diff
@@ -76,7 +76,7 @@ public static void tearDown() {
      */
     @Test
     public void canCreateApiKeyForSelf() {
-        assumeTrue(getBooleanEnvVar("RUN_E2E"));
+        assumeTrue(isEndToEnd);
```

Comment:
Do we still need this assertions? It is already checked in the @BeforeAll code.

Reply:
good catch.